.github/CHANGELOG.md

```diff
@@ -2,6 +2,51 @@
 
 <h3>New features since last release</h3>
 
+* Adds a new optimizer `qml.ShotAdaptiveOptimizer`, a gradient-descent optimizer where
+  the shot rate is adaptively calculated using the variances of the parameter-shift gradient.
+  [(#1139)](https://github.com/PennyLaneAI/pennylane/pull/1139)
+
+  By keeping a running average of the parameter-shift gradient and the *variance* of the
+  parameter-shift gradient, this optimizer frugally distributes a shot budget across the partial
+  derivatives of each parameter.
+
+  In addition, if computing the expectation value of a Hamiltonian, weighted random sampling can be
+  used to further distribute the shot budget across the local terms from which the Hamiltonian is
+  constructed.
+
+  This optimizer is based on both the [iCANS1](https://quantum-journal.org/papers/q-2020-05-11-263)
+  and [Rosalin](https://arxiv.org/abs/2004.06252) shot adaptive optimizers.
+
+  ```pycon
+  >>> coeffs = [2, 4, -1, 5, 2]
+  >>> obs = [
+  ...   qml.PauliX(1),
+  ...   qml.PauliZ(1),
+  ...   qml.PauliX(0) @ qml.PauliX(1),
+  ...   qml.PauliY(0) @ qml.PauliY(1),
+  ...   qml.PauliZ(0) @ qml.PauliZ(1)
+  ... ]
+  >>> H = qml.Hamiltonian(coeffs, obs)
+  >>> dev = qml.device("default.qubit", wires=2, analytic=False)
```

Comment:
By the time this is merged in, will users still need to specify `analytic` as a keyword to device?

Reply:
Nope, good catch - have updated it to remove the `analytic` argument